src/bot.coffee

```diff
@@ -199,4 +204,32 @@ class SlackBot extends Adapter
     item_user = @client.rtm.dataStore.getUserById(item_user)
     @receive new ReactionMessage(type, user, reaction, item_user, item, event_ts)
 
+  loadUsers: (err, res) =>
+    if err || !res.ok
+      @robot.logger.error "Can't fetch users"
+      return
+
+    for id, user of res.members
```

Comment:
wha you are getting here is the JavaScript `for...of`, but the usage looks confusing because it seems as though you are trying to do a comprehension and pull out properties `id` and `user` of `members`. since `members` is actually an array and not a normal object, you will get `id` that is the index in the array and `user` that is the value. this happens to work, but it doesn't read clearly.

I suggest:

`@userChange member for member in res.members`